main.tf

```diff
@@ -65,7 +67,7 @@ resource "aws_lb" "main" {
   access_logs {
     enabled = true
     bucket  = var.logs_s3_bucket
-    prefix  = "alb/${var.name}-${var.environment}"
+    prefix  = "alb/AWSLogs/${data.aws_caller_identity.current.account_id}/${var.name}-${var.environment}"
```

Comment:
I think this will tell the ALB to log to `alb/AWSLogs/123456789012/${var.name}-${var.environment/alb/AWSLogs/123456789012`

The rough edge here is the default policy for alb logs is `alb/AWSLogs/123456789012` and this module tries to log to a different prefix `alb/${var.name}-${var.environment}/AWSLogs/123456789012`. It's not great, but the fix is to modify the alb_logs_prefix in the aws-logs module to be something like 

```
 alb_logs_prefixes = [
   "alb/${var.test_name}-${local.environment}"
 ]
```

Reply:
ahh... you're right. I'll go look at the aws-logs module